demo/package.json

```diff
@@ -64,6 +64,7 @@
 		"react-hook-form": "^7.0.0",
 		"react-json-view": "^1.21.1",
 		"react-markdown": "^7.0.1",
+		"react-monaco-editor": "^0.47.0",
```

Comment:
@titkov49 Is this library necessary in demo? I mean now we have it isolated in asab-library ui, or not?